htdocs/web_portal/views/downtime/downtimes_overview.php

```diff
@@ -25,11 +26,17 @@
         </div>
         <div style="float: left;">
             <h1 style="float: left; margin-left: 0em; padding-bottom: 0.0em;">
-                   Overview of Current and Planned Downtimes
+                   Downtimes
```

Comment:
Can you revert this line? I think it's clearer as before.

```suggestion
                   Overview of Current and Planned Downtimes
```

Reply:
Reverted.